src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectPaperTableRepo.kt

```diff
@@ -28,6 +29,18 @@ interface IProjectPaperTableRepo {
      */
     suspend fun getProjectPaperById(id: UUID): ProjectPaper
 
+    /**
+     * Retrieves a specific project paper associated with a given project and relative ID.
+     *
+     * This method searches for a project paper based on the provided project ID and
+     * relative ID (local paper ID). Throws a [NotFoundException] if no matching project paper is found.
+     *
+     * @param projectId The unique identifier of the project to which the project paper belongs.
+     * @param relativeId The local paper ID of the project paper to retrieve.
+     * @return The [ProjectPaper] matching the given project ID and relative ID.
+     */
```

Comment:
```suggestion
    /**
     * Retrieves a project paper by its relative ID in a project.
     *
     * Therefore, this method searches for a project paper in the project with the [projectId] with
     * the [relativeId] (`localPaperId`).
     *
     * @param projectId The unique identifier of the project to which the project paper belongs.
     * @param relativeId The local paper ID of the project paper in the project.
     * @return The [ProjectPaper] matching the given project ID and relative ID.
     * @throws A [NotFoundException] if no matching project paper is found.
     */
```

What do you think of this suggestion? 
If a method is throwing an error and you want to declare this, you can use the "@throws" keyword and if you link to a certain field of the object (local_id) which is called different than the parameter but is actually the same, then I would use the correct name.

Reply:
This is a good one. Tbh, I autogenerated it and did not pay attention on the `throws` keyword. Good call.

Reply:
Next time, please review the autogenerated comments, but anyways I want to discuss how we deal with the "@throws" keyword and consistency issues in the comments and tests in the next sprint meeting.